### DISCUSSION TITLE
Why does a discussion disappear with a 404 page not found.

### DISCUSSION THREAD

**Discussion by Mattmjs** [aclap-dev/video-downloadhelper#1393] [Help and Q&A]
### Your question

I was trying to respond to this discussion https://github.com/aclap-dev/video-downloadhelper/discussions/1387
It didn't accept the response so I reloaded it only to find a page not found.

### Addon & CoApp version and details

Not needed.

**Comment by GrampaWildWilly** [comment]
I've had that happen to me a couple of times as well.  It's weird.  I don't know why it happens.  Maybe some GitHub bot thinks the discussion somehow violates community standards.  Maybe somebody reported the discussion & a GitHub employee made the determination.  It's certainly disconcerting when it happens.  It has always seemed to me to be an inappropriate removal.  I don't know what to do about it.  Maybe open a discussion in the GitHub support area.
